test/test_kl.jl

```diff
@@ -83,4 +83,4 @@ test_beta_kl_value()
 test_categorical_kl_value()
 test_diagmvn_mvn_kl_value()
 test_gaussian_kl_value()
-test_subtract_kl()
+@test_skip test_subtract_kl()
```

Comment:
Hard coding is the way to go if you happen to have some known correct values floating around, otherwise it should be sufficient to test against Calculus.jl's finite difference approximations (you'll have to use `@test_approx_eq_eps` with some appropriate epsilon value due to the approximation error). These derivative calculations already use both ReverseDiff and ForwardDiff so testing them against each other wouldn't be too meaningful.